hibernate-core/src/main/java/org/hibernate/dialect/CockroachDialect.java

```diff
@@ -87,6 +87,9 @@ public CockroachDialect(int version) {
 		registerColumnType( Types.NCLOB, "string" );
 
 		registerColumnType( Types.JAVA_OBJECT, "json" );
+
+		//register geometry type
+		registerColumnType( 5432, "geometry" );
```

Comment:
Agreed. But is it wise to do this in this PR. Should we not open a separate PR for this since it is not limited to Spatial. See our discussion on this from July 28.

Reply:
Sure, you can do this in a follow up PR. Just add yourself a TODO item for this ;)